Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,107 @@
+//zad 1
+// const reverseText = (string) => {
+//     let newString = "";
+//     for (let i = string.length - 1; i >= 0; i--) {
+//       newString += string[i];
+//     }
+//     return newString
+// };
+
+// const reverseText = (string) =>{
+//     return string.split("").reverse().join("");
+
+// }
+
+// reverseText("infoShare"); // => erahSofni
+// reverseText("iSA"); // => ASi
+// reverseText("Adam"); // => madA
+
+//zad 2
+
+// const splitString = (string) => {
+//     const newString = string.split(" ");
+//     let firstWord = newString[0];
+//     let secondWord = newString[1];
+//     let firstLetterA = firstWord[0];
+//     let firstLetterB = secondWord[0];
+//     return `${firstLetterB+firstWord.slice(1)} ${firstLetterA+secondWord.slice(1)}`;
+
+//      console.log(firstWord);
+//      console.log(secondWord);
+//      console.log(firstletterA);
+//      console.log(firstletterB);
+
+//    }
+
+//    console.log(splitString("Adam Kocinski"))
+
+// zad 3
+// let arr1 = [1,2,3];
+// let arr2 = [4,2,8];
+// function multiDiff (a,b) {
+//     const multi1 = a.reduce((a,b)=>a*b);
+//     const multi2 = arr2.reduce((a,b)=>a*b);
```

Comment:
nie korzystasz tutaj z argumentu "b" funkcji, tylko ze zmiennej którą sobie zdefiniowałeś wyżej. Teraz wychodzi dobrze, bo do tej funkcji wstrzykujesz i tak tylko te zmienne, no ale jest to błąd :) i chyba wiem z kim byłeś w grupie :D

Reply:
masz racje powinno byc b.reduce